m03_acos/create_acos_exercises.rst

```diff
@@ -0,0 +1,454 @@
+4. Change the name of the module from ``DragandDropExample`` to whichever name you prefer for your exercise (e.g. ``DragandDropDataManagement``). 
+
+After making all the changes mentioned above, your ``coffee.index`` should looks something like :ref:`the image below <acos-changes>`. Notice that in this example we did not change the meta-information, but you should change it.
+
+.. figure:: /images/gallery/index.coffee.png
+  :name: acos-changes
+  :width: 95%
+  :align: center
+
+  Diff view between the original file (left) and the edited one(right).
+
+Now that you have edited the some of the source files, you can proceed to install the ``coffeescript`` module and compile the ``index.coffee`` file. Therefore, you should run the following command in your project directory.
+
+.. code-block:: bash
+
+  npm install
+
+Verify that a ``node_modules`` folder and the ``index.js`` file are created. If the ``index.js`` file is not created, you should run the following command in your project directory.
+
+.. code-block:: bash
+
+  npm run prepare
+
+Create your own Acos exercise
+.............................
+Now that your project has been set up, it is time to start creating your own exercise. In this chapter, we will create a data management exercise so that you can follow step-by-step the process of creating Acos exercise. Remember that all the exercises you want to create should be located inside the ``exercises`` folder. You can also group them and organise them in subfolders. For example, in the current ``exercises`` folder, there is an ``images`` folder and an ``articles`` folder containing specific exercises. 
+
+For now, let's delete all the current examples and the related static files. Once, the ``exercises`` folder is empty we will start creating a new **XML** and **JSON**. In this case, we will call the exercise ``data_structure``. Therefore, we will have two files. One called ``data_structure.xml`` and the second one called ``data_structure.json``. Remember, that the name of those files should not contain hyphens and it is preferred to use an underscore to separate words in the name of the files. Finally, we will add our images to the ``static`` folder, and as an end result, we should have a project structure like the one :ref:`shown below <acos-exercise-folder-structure>`.
+
+.. code-block:: bash
+  :name: acos-exercise-folder-structure
+  :caption: The structure of the project should looks like this.
+  :emphasize-lines: 3,4,11-21
+
+  .
+  ├── exercises
+  │   ├── data_structure.json
+  │   └── data_structure.xml
+  ├── index.coffee
+  ├── index.js
+  ├── LICENSE
+  ├── node_modules
+  ├── package-lock.json
+  ├── README.md
+  └── static
+      ├── data_folder.png
+      ├── date_folder.png
+      ├── exp_folder.png
+      ├── file.png
+      ├── link_folder2.png
+      ├── link_folder.png
+      ├── project_folder.png
+      ├── readme.png
+      ├── scripts.png
+      └── src_folder.png
```

Comment:
For some reason, this block is very wide even when the window is slim, seems like it doesn't shrink at all.

Reply:
Yes, this is a problem with the Aplus theme, but it will be fixed in Aalto-LeTech/a-plus-rst-tools#88